packages/installer/installer.sh

```diff
@@ -18,11 +18,11 @@ umount_target() {
         umount ${TARGET}/oem || true
         umount ${TARGET}/usr/local || true
         umount ${TARGET}/proc || true
-        umount ${TARGET}/dev || true
-        umount ${TARGET}/sys || true
+        umount -R ${TARGET}/dev || true
+        umount -R ${TARGET}/sys || true
         umount ${TARGET}/boot/efi || true
         umount ${TARGET}/boot/grub2 || true
-        umount ${TARGET} || true
+        umount ${TARGET}
```

Comment:
Probably we could consider doing only a single `umount --recursive ${TARGET}` to unmount everything. In the past I also noted that in some cases the lazy umount helps so probably:

```
umount -l --recursive ${TARGET}
```

does the job in a single command and there is not need to umount manually all of them.

Reply:
I did avoided to use `-l` here because when we umount $TARGET we want to be sure  it's not there, because its the image file that later on gets copied (hence the fix here). If something else umounts it later then it would be too late, because we copy it over in the STATE partition